presto-iceberg/src/main/java/io/prestosql/plugin/iceberg/InternalIcebergConnectorFactory.java

```diff
@@ -68,7 +68,7 @@ public static Connector createConnector(String catalogName, Map<String, String>
                         binder.bind(NodeManager.class).toInstance(context.getNodeManager());
                         binder.bind(TypeManager.class).toInstance(context.getTypeManager());
                         binder.bind(PageIndexerFactory.class).toInstance(context.getPageIndexerFactory());
-                        binder.bind(HiveCatalogName.class).toInstance(new HiveCatalogName(catalogName));
+                        binder.bind(CatalogName.class).toInstance(new CatalogName(catalogName));
```

Comment:
What about extracting `CatalogNameModule`?

Reply:
that would call for not using `catalogName` directly, only via guice. beyond scope.